### PR TITLE
[FEAT] lesson 8 task 16

### PR DESCRIPTION
**Pull request by heilrake**

### CODE REVIEW

File: src/lesson/Lesson8/Task16/index.jsx [hunk 1]
```diff
@@ -16,44 +17,71 @@ const Task16 = () => {
...
+  const handleChangeAnswer = (questionIndex, answerId) => {
+    const copyQuestions = [...questions];
+
+    copyQuestions[questionIndex].answer = answerId;
+    setQuestions(copyQuestions);
+  };
+
+  const getStatusClassName = (rightAnswer, currentAnswer, answerId) => {
+    if (rightAnswer === currentAnswer && currentAnswer === answerId) {
+      return "correct"
```
Comment (daviddubliakov) [inline src/lesson/Lesson8/Task16/index.jsx]: Add ":" in the end of line

File: src/lesson/Lesson8/Task16/index.jsx [hunk 2]
```diff
@@ -16,44 +17,71 @@ const Task16 = () => {
...
+    }
+
+    if (currentAnswer === answerId) {
+      return "false"
```
Comment (daviddubliakov) [inline src/lesson/Lesson8/Task16/index.jsx]: Add ":" in the end of line

File: src/lesson/Lesson8/Task16/index.jsx [hunk 3]
```diff
@@ -16,44 +17,71 @@ const Task16 = () => {
...
+    }
+
+    return "default"
```
Comment (daviddubliakov) [inline src/lesson/Lesson8/Task16/index.jsx]: Add ":" in the end of line

**Review by daviddubliakov — CHANGES_REQUESTED** [review]
Please check my comments

File: src/lesson/Lesson8/Task16/index.jsx (hunk 1 above)
Comment (AI-Avanger) [reply src/lesson/Lesson8/Task16/index.jsx]: Fixed

File: src/lesson/Lesson8/Task16/index.jsx (hunk 2 above)
Comment (AI-Avanger) [reply src/lesson/Lesson8/Task16/index.jsx]: Fixed

File: src/lesson/Lesson8/Task16/index.jsx (hunk 3 above)
Comment (AI-Avanger) [reply src/lesson/Lesson8/Task16/index.jsx]: Fixed

**Review by daviddubliakov — APPROVED** [review]

**Pull request merged**